ztp/gitops-subscriptions/argocd/deployment/app-project.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: default
+  namespace: openshift-gitops
+spec:
+  clusterResourceWhitelist:
+  - group: '*'
+    kind: '*'
+  destinations:
+  - namespace: '*'
+    server: '*'
+  namespaceResourceWhitelist:
+  - group: '*'
+    kind: '*'
+  sourceRepos:
+  - '*'
```

Comment:
I changed to the permissions we actually used for the demo

Reply:
Yes, we may need to have 2 different AppProjects one for clusetrs-app and other for common, groups and sites Apps  but let's merge your change first and then we can update.

Reply:
We should open a story to revisit these permissions and reduce them down to what is actually required.

Reply:
https://issues.redhat.com/browse/CNF-2727